tools/BuildPackagesTask/packages.config

```diff
@@ -0,0 +1,11 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
there is one config file per project. One for test and one for the main project. I restructured the folder so it matches the structure followed in the repo.